pkg/controllers/vrpq/query_reconciler.go

```diff
@@ -134,3 +179,37 @@ func (q *QueryReconciler) runListRestorePoints(ctx context.Context, _ *ctrl.Requ
 	return vrpqstatus.UpdateConditionAndState(ctx, q.VRec.Client, q.VRec.Log, q.Vrpq,
 		v1.MakeCondition(vapi.QueryComplete, metav1.ConditionTrue, "Completed"), stateSuccessQuery)
 }
+
+// makeDispatcher will create a Dispatcher object based on the feature flags set.
+func (q *QueryReconciler) makeDispatcher(log logr.Logger, vdb *v1.VerticaDB, prunner cmds.PodRunner,
+	passwd string) vadmin.Dispatcher {
+	if vmeta.UseVClusterOps(vdb.Annotations) {
+		return vadmin.MakeVClusterOps(log, vdb, q.VRec.GetClient(), passwd, q.VRec, vadmin.SetupVClusterOps)
```

Comment:
The password isn't needed since we our API is going to talk strictly to the NMA. We should make the password field a pointer so that we will not that its not used rather than a dbadmin account without a password being set.